src/array_analysis.js

```diff
@@ -0,0 +1,8 @@
+const analyze = (arr) => ({
+  avg: arr.reduce((a, b) => a + b) / arr.length,
```

Comment:
- [ ] [OPTIONAL] What would happen if the length was 0? Can you handle the case and add a test for it?